WoWsShipBuilder.Core/DataContainers/Projectiles/DepthChargeDataContainer.cs

```diff
@@ -5,7 +5,7 @@
 using WoWsShipBuilder.Core.Extensions;
 using WoWsShipBuilder.Core.Services;
 using WoWsShipBuilder.DataElements.DataElementAttributes;
-using WoWsShipBuilder.DataStructures;
+using WoWsShipBuilder.DataStructures.Projectile;
 
 namespace WoWsShipBuilder.Core.DataContainers
 {
```

Comment:
If possible, i think we should use a placeholder, like "N/A", instead of disabling the entire property. DC missing localizations are probably new ships ones, so a limited case.

Reply:
Property removed because it seems like there is no actual localized name for them